shell/server/backup.js

```diff
@@ -183,7 +183,7 @@ Router.map(function () {
       }
 
       let started = false;
-      const filename = (token.name.replace(/["\n]/g, "") || "backup") + ".zip";
+      const filename = encodeURIComponent(token.name || "backup") + ".zip";
```

Comment:
Might I suggest naming the variable "encodedFilename" or similar, so the uses of it below look less suspicious?